show/main.py

```diff
@@ -1407,24 +1425,24 @@ def runningconfiguration():
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def all(verbose):
     """Show full running configuration"""
-    cmd = ['sonic-cfggen', '-d', '--print-data']
-    stdout, rc = get_cmd_output(cmd)
-    if rc:
-        click.echo("Failed to get cmd output '{}':rc {}".format(cmd, rc))
-        raise click.Abort()
 
-    try:
-        output = json.loads(stdout)
-    except JSONDecodeError as e:
-        click.echo("Failed to load output '{}':{}".format(cmd, e))
-        raise click.Abort()
+    if multi_asic.is_multi_asic():
+        output = {}
+        # In multiaisc, the namespace is changed to localhost by design
+        output['localhost'] = get_config_json_by_namespace(multi_asic.DEFAULT_NAMESPACE)
+
+        ns_list = multi_asic.get_namespace_list()
+        for ns in ns_list:
+            output[ns] = get_config_json_by_namespace(ns)
+    else:
+        output = get_config_json_by_namespace(None)
```

Comment:
>output = get_config_json_by_namespace(None) [](http://example.com/codeflow?start=8&length=43)

For single asic, we can unify the logic like:
```
output['localhost'] = get_config_json_by_namespace(multi_asic.DEFAULT_NAMESPACE)
```
and later on return only `output['localhost']`

Reply:
As I said, single-asic / multi-asic logic may be unified. Just treat DEFAULT_NAMESPACE as one of the namespace, all other logic are the same.

Reply:
This is outdated change. Please check the lastest change.